cli_reference/get_started_cli.adoc

```diff
@@ -135,9 +178,16 @@ Now using project 'project03'.
 $ osc project
 Using project 'project03'.
 ----
+====
 
-The same way `git status` provides a general overview of the working tree of a local git repository, we provide a command that shows a high level overview of the project currently in use, with its components and their relationships. This command takes no argument. Sample output of the `osc status` command:
+The same way `git status` provides a general overview of the working tree of a
+local git repository, the `osc status` command shows a high level overview of
+the project currently in use, with its components and their relationships. This
+command takes no argument:
 
```

Comment:
Why do we have this statement - "The same way `git status` provides....."? I don't see any relevance of this statement in this context.

[Suggestion]
The `osc status` command shows a high level overview of the project currently in use, with its components and their relationships, as shown in the following example.

[Example]
screen output